src/Servers/Kestrel/Core/src/Internal/Infrastructure/KestrelEventSource.cs

```diff
@@ -220,7 +220,7 @@ public void TlsHandshakeFailed(string connectionId)
         }
 
         [MethodImpl(MethodImplOptions.NoInlining)]
-        [Event(11, Level = EventLevel.Informational)]
+        [Event(11, Level = EventLevel.LogAlways)]
```

Comment:
Why `LogAlways`?

Reply:
Payload is small enough to not cause any damage